components/page/FilterBar/index.js

```diff
@@ -30,133 +22,36 @@ const FilterBar = ({ filterType, setLiveRegionMessage }) => {
-              aria-controls="filter-dropdown"
-            >
-              <Styled.ToggleIcon />
-              <span>Filter</span>
-            </Styled.ToggleButton>
-            <Styled.ToggleDropdown id="filter-dropdown" $opened={filterOpen}>
-              <li>
-                <MixedLink url={asPath} params={{ filter: "" }}>
-                  <T i18nKey={`filters.all`} />
-                </MixedLink>
-              </li>
-              {filterItems.map((item, i) => {
-                const active = params?.filter?.includes(item.id);
-
-                return (
-                  <li key={i}>
-                    <MixedLink
-                      className={active ? "active" : ""}
-                      url={asPath}
-                      params={{ filter: item.id }}
-                    >
-                      {item.title}
-                    </MixedLink>
-                  </li>
-                );
-              })}
-            </Styled.ToggleDropdown>
-          </div>
-        )}
-        {hasSort && (
-          <div>
-            <Styled.ToggleButton
-              onClick={handleSort}
-              aria-expanded={sortOpen}
-              aria-controls="sort-dropdown"
-            >
-              <Styled.SortToggleIcon id="sort" />
-              <span>{t(`filters.sort`)}</span>
-            </Styled.ToggleButton>
-            <Styled.ToggleDropdown id="sort-dropdown" $opened={sortOpen}>
-              {sortItems.map((item, i) => {
-                const active = params?.sort?.includes(item.slug);
-
-                return (
-                  <li key={i}>
-                    <MixedLink
-                      className={active ? "active" : ""}
-                      url={asPath}
-                      params={{ sort: item.slug }}
-                    >
-                      {item.title}
-                    </MixedLink>
-                  </li>
-                );
-              })}
-            </Styled.ToggleDropdown>
-          </div>
-        )}
-        {hasSearch && (
-          <Styled.FilterSearch onSubmit={handleSubmit}>
-            <button type="submit">
-              <IconComposer icon="search" />
-              <span className="a-hidden">{t("submit-search")}</span>
-            </button>
-            <label htmlFor="filterSearchInput" className="a-hidden">
-              {t("search-filter")}
-            </label>
-            <input
-              id="filterSearchInput"
-              type="search"
-              placeholder={t("search-filter-placeholder")}
-              value={searchText}
-              onChange={(e) => handleChange(e.target.value)}
-            />
-          </Styled.FilterSearch>
-        )}
-        <Styled.Clear onClick={handleReset} data-cy="clear">
-          <IconComposer icon="cancel" />
-          {t(`search-clear`)}
-        </Styled.Clear>
-      </Styled.FilterGrid>
-    </Styled.FilterNav>
+    <Filters hasSearch={hasSearch} onFiltersCleared={handleReset}>
```

Comment:
a lot of the pieces in here were extracted to create the new Search, ClearFilters, and FilterDropdownList components